builds/e2e/connectivity.yaml

```diff
@@ -286,6 +303,7 @@ jobs:
           testStartDelay: '$(testStartDelay.arm32)'
           networkController.frequencies: '$(testrun.network.frequencies)'
           networkController.runProfile: '$(testrun.network.runProfile)'
+          networkController.description: '$(testrun.network.description)'
```

Comment:
Since network controller frequencies and run profile is passed to module for test run and those information already reported in TestInfo in https://github.com/Azure/iotedge/blob/cf39ecd6298892533d6525520dac6a0e07016cb0/test/connectivity/modules/NetworkController/Program.cs#L84

for description of network controller, it is just used in Test dashboard;  therefore this change should be in Test dashboard.  Also for any information already passed to module, it should let the module to report to TRC using TestInfoResult like what did in NetworkController.  After review, testBuildNumber should not be defined in TestInfo; as it is already well-defined in TestSummary which test dashboard logic should extract from there.  Similarly for any information passed to connectivityTest.sh, those field should append to TestInfo inside connectivityTest.sh instead of in connectivity-deploy.yaml, for example, customEdgeAgentImage.

Let me know if you want to chat about this.